judge/runguard.cc

```diff
@@ -363,7 +365,8 @@ Run COMMAND with restrictions.\n\
   -s, --streamsize=SIZE  truncate COMMAND stdout/stderr streams at SIZE kB\n\
   -E, --environment      preserve environment variables (default only PATH)\n\
   -V, --variable         add additional environment variables\n\
-                           (in form KEY=VALUE;KEY2=VALUE2)\n\
+                           (in form KEY=VALUE;KEY2=VALUE2); may be passed\n\
+			   multiple times\n\
```

Comment:
Indentation is off here because of tabs.

Reply:
Done